ui/tests/integration/components/project-input-variables-list-test.ts

```diff
@@ -26,6 +27,11 @@ module('Integration | Component | project-input-variables-list', function (hooks
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
+  hooks.beforeEach(function (this: TestContext) {
+    // We have to register any types we expect to use in this component
+    this.owner.lookup('service:flash-messages').registerTypes(['success', 'error']);
+  });
```

Comment:
I was surprised this is needed, but the README says so:
https://github.com/poteto/ember-cli-flash#acceptance--integration-tests